backend/src/main/java/com/example/subjecthub/entity/Assessment.java

```diff
@@ -44,7 +54,9 @@
     private String length;
 
     @Column(nullable = false)
-    private AssessmentType type;
+    //18092017
+    //private AssessmentType type;
+    private Integer type;
```

Comment:
We should keep the safety of using an enum. Can you please change type to be an enum.